### DISCUSSION TITLE
How to set the share by mail on the footer part of the website.

### DISCUSSION THREAD

**Discussion by fmorfini** [sylhare/Type-on-Strap#456] [Q&A] [answered]
Hi,

I'm having issues with the email/envelope icon in the footer. When I click on it, I would expect to generate a new email message either on a local email app (e.g., I'm using mail.app from a Mac) or some other way, but instead a new blank tab opens up on the web browser. I'm using the latest google chrome version 127.0.6533.100 (Official Build) (x86_64). 

This is my [repo](https://github.com/fmorfini/fmorfini.github.io) and the [website](https://fmorfini.github.io/). Note, that I've currently removed the envelope icon to avoid issues on the live website, however the issue I was having can be reproduced by changing the following info on these fils:

`/_data/icons_builder.yml` (unchanged compared to the template):
```
mail:
  pre: "mailto:"
  icon: "fas fa-envelope"
```

`/_data/social.yml`:
```
share:
  email: true
```

`/_data/icons.yml`:
```
mail: f.morfini.work@gmail.com
```

Also, I tried to get around this issue by adding the email as plain text in the footer. I don't know if this is a separate issue, but the text I entered for `footer_text`in the `_config.yml` file isn't displaying in the website:

```
footer_text: >
  f.morfini.work@gmail.com
# Powered by <a href="https://jekyllrb.com/">Jekyll</a> with <a href="https://github.com/sylhare/Type-on-Strap">Type on Strap</a>
```
it does show up when I change the `_includes/default/footer.liquid`:
```
        {{ site.footer_text | default: "f.morfini@gmail.com" }}</p>
```

Any suggestions? 
Thank you for putting out this great template!

**Comment by sylhare** [comment]
@fmorfini so the email share in  `/_data/social.yml` is actually for the blog post:
![image](https://github.com/user-attachments/assets/4c34fafb-8b3d-47e0-837e-278b2a8c074d)

To get it as part of the footer you can edit `/_data/icons.yml`
![image](https://github.com/user-attachments/assets/cd477b47-d19d-41b1-b4bb-af578355c55c)

And then set the mail as:
```yml
mail: f.morfini.work@gmail.com
```
It's independent from the `footer_text`.

**Reply by sylhare** [reply]
@fmorfini I converted it to a discussion as it's a great question. You can click on `Mark as answer` on the response if it works and that helped you 👍

**Comment by fmorfini** [comment]
Thanks @sylhare ! ah I see. I'm not using the share option so that explains part of it. The email icon I'm having issues with is the one that appears in the footer (see screenshot). I've added back the envelop icon in my website and it should online now. When I click on it a new blank page appears 
<img width="420" alt="Screenshot 2024-08-08 at 6 37 39 PM" src="https://github.com/user-attachments/assets/aba8d2ef-e09a-48fd-ada0-8661149c5ff3">

**Reply by sylhare** [reply]
I tried on your website and it seems to work now, when I click on 
![image](https://github.com/user-attachments/assets/d2e7ba08-83c4-422e-81a5-b7b820e807b5) the mail software that I use does open up (`Mail` app on MacOS).

**Reply by fmorfini** [reply]
thanks for checking it out. mmm I see. I wonder if it's a specific to my local machine or OS (thou it's recent: I'm on a Mac running Ventura 13.6.7 and use Mail.app as well). this is what i get when i click on it. All the other icons (github, scholar, etc) work as expected

<img width="662" alt="Screenshot 2024-08-08 at 7 00 03 PM" src="https://github.com/user-attachments/assets/374a1c88-2718-4a62-b56e-a92e38956af0">

**Reply by sylhare** [reply, marked as answer]
It could be a security setting that might be off on your browser, that would explain why it opens a new blank page instead of opening the default email client.

**Reply by fmorfini** [reply]
makes sense. thanks for your inputs!
